README.md

```diff
@@ -1 +1,41 @@
-# Portfolio-setup-and-mobile-first
\ No newline at end of file
+# Portfolio-setup-and-mobile-first
+# Portfolio-setup-and-mobile-first![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Portfolio-setup-and-mobile-first.
+
+
+## Getting Started
+
+**This is the second project created on microverse education by me.**
```

Comment:
[ ] You did a great job documenting your project! 📜 However it would be even great if you add a Getting started section describing the required steps others need to follow in order to get a local copy of your project up and running!